src/main/resources/templates/db_scripts/data-migration/invoices_table.sql

```diff
@@ -1,3 +1,9 @@
 CREATE SEQUENCE IF NOT EXISTS ${myuniversity}_${mymodule}.invoice_number NO MAXVALUE START WITH 10000 CACHE 1 NO CYCLE;
 GRANT USAGE ON SEQUENCE ${myuniversity}_${mymodule}.invoice_number TO ${myuniversity}_${mymodule};
 CREATE UNIQUE INDEX IF NOT EXISTS invoices_invoice_number_unique_idx ON ${myuniversity}_${mymodule}.invoices ((jsonb->>'invoiceNumber'));
+
+UPDATE ${myuniversity}_${mymodule}.invoices
```

Comment:
Now script section supports snippetPath, so as an improvement, you can put migration scripts into a separate file.
https://github.com/folio-org/mod-finance-storage/blob/49bf1480749e40fdb516d3f0a0057f1c8a1a3197/src/main/resources/templates/db_scripts/schema.json#L2-L8

Reply:
ah, cool... I thought it was crappy that we needed to do it this way so I'm glad there's a better way!